projects/ui/src/shim.cds-core.scss

```
@@ -499,68 +499,128 @@
+  --clr-badge-warning-bg-color: var(--cds-alias-utility-yellow); // var(--clr-color-warning-1000);
   --clr-badge-warning-color: var(--cds-global-color-black); // var(--clr-color-neutral-0);
-  --clr-badge-danger-bg-color: var(--cds-alias-status-danger-shade); // var(--clr-color-danger-900);
-  --clr-badge-danger-color: var(--cds-global-color-white); // var(--clr-color-neutral-0);
-
-  --clr-badge-gray-bg-color: var(--cds-global-color-gray-600); // var(--clr-color-neutral-600);
-  --clr-badge-gray-color: var(--cds-global-color-white); // var(--clr-badge-font-color-light);
-  --clr-badge-purple-bg-color: var(--cds-global-color-violet-500); // var(--clr-color-secondary-action-500);
-  --clr-badge-purple-color: var(--cds-global-color-white); // var(--clr-badge-font-color-light);
-  --clr-badge-blue-bg-color: var(--cds-global-color-blue-800); // var(--clr-color-action-800);
-  --clr-badge-blue-color: var(--cds-global-color-white); // var(--clr-badge-font-color-light);
-  --clr-badge-orange-bg-color: var(--cds-global-color-tangerine-400); // var(--clr-color-warning-1000);
+  --clr-badge-danger-bg-color: var(--cds-alias-utility-red); // var(--clr-color-danger-900);
+  --clr-badge-danger-color: var(--cds-alias-typography-color-100, var(--cds-global-typography-color-100)); // var(--clr-color-neutral-0);
+
+  --clr-badge-gray-bg-color: var(--cds-alias-utility-gray); // var(--clr-color-neutral-600);
+  --clr-badge-gray-color: var(--cds-alias-typography-color-100, var(--cds-global-typography-color-100));
+  --clr-badge-purple-bg-color: var(--cds-alias-utility-violet); // var(--clr-color-secondary-action-500);
+  --clr-badge-purple-color: var(--cds-alias-typography-color-100, var(--cds-global-typography-color-100)); // var(--clr-badge-font-color-light);
+  --clr-badge-blue-bg-color: var(--cds-alias-utility-blue); // var(--clr-color-action-800);
+  --clr-badge-blue-color: var(--cds-alias-typography-color-100, var(--cds-global-typography-color-100));
+  --clr-badge-orange-bg-color: var(--cds-alias-utility-tangerine); // var(--clr-color-warning-1000);
   --clr-badge-orange-color: var(--cds-global-color-black); // var(--clr-color-neutral-0);
-  --clr-badge-light-blue-bg-color: var(--cds-global-color-blue-300); // var(--clr-color-action-500);
-  --clr-badge-light-blue-color: var(--cds-global-color-black); // var(--clr-color-neutral-0);
+  --clr-badge-light-blue-bg-color: var(--cds-alias-utility-lightblue); // var(--clr-color-action-500);
+  --clr-badge-light-blue-color: var(--cds-alias-typography-color-500, var(-cds-global-typography-color-500)); // var(--clr-color-neutral-0);
+
+  --clr-badge-vertical-padding: var(--cds-global-space-2); // 2px
+  --clr-badge-horizontal-padding: var(--cds-global-space-3); // 4px
+  --clr-badge-height: var(--cds-global-space-7); // 16px
+  --clr-badge-min-width: var(--cds-global-space-7); // 16px
+  --clr-badge-border-radius: calc(2 * var(--cds-alias-object-border-radius-100));
```

Figma suggest using `--cds-alias-object-border-radius-300 / 8px`, which is a conflicting statement. Can you check with the designers what is expected?
My understanding is `radius-300` is `50%` from total `16px` height, but I might be wrong in my assumption.

```suggestion
  --clr-badge-border-radius: var(--cds-alias-object-border-radius-300);
```